rfc/2022/2022-09-15_Design-System-Team_Migrate-vets-website-off-of-formation.md

```diff
@@ -0,0 +1,106 @@
+
+This will have a performance benefit as well, since an audit of that particular CSS class would reveal that the `search-result-tag` CSS class isn't used in application code at all, so we are shipping unused CSS that should be deleted. As we begin to rely more on utility classes and developers write less CSS, this type of thing will happen less often.
+
+#### Prune unneeded pieces of formation
+
+This is somewhat covered in the "Remove component/module CSS" section, but as we make progress with the migration we will routinely run searches to see if there is any remaining code which relies on a given CSS class that is part of the migration. If there isn't, we can remove `formation` imports from `vets-website` which will let us get a better picture of which classes are actually in use. This will also improve performance by sending less CSS.
+
+#### Identify styles/classes what have no component or utility available
+
+As we audit and migrate each app we may find that there is custom styling happening which cannot be replaced using current components or utilities. When this happens we will make a note of the files & styles and consider if they should be brought into the DS in one way or another.
+
+## Risks
+
+### Breaking the appearance of a page or application
+
+The goal is to reduce this risk as much as possible by making small incremental changes that are tested each step of the way. Relying on ESLint rules to ["stop the bleeding"](https://lethain.com/migrations/) should prevent new code from being added which uses older classes that have been replaced.
+
+## Alternatives
+
+### Moving directly to USWDS3 instead of the VADS CSS library
+
+The problem with this is that it would leave us in the same situation wer're in now where future changes are more difficult.
+By limiting the available CSS classes in `vets-website`, we will have an easier time making upgrades.
+
+Part of the problem we're dealing with now is that `usa-*` classes are used throughout `vets-website`, and these are corresponding to USWDS1 styles. USWDS3 uses many of the same names like `usa-alert` which means that a drop-in replacement/upgrade is impossible to do with confidence that it won't break existing styles. Putting the VADS between `vets-website` and direct use of USWDS means that the DST will be able to manage these kinds of version upgrades in the future while minimizing risk in `vets-website`.
+
+### Sticking with formation & USWDS 1
+
+This will lead to further version skew and make it harder to adopt USWDS fixes and changes.
+
+## Diagrams
```

Comment:
I can try and find some time to visualize this but I don't know that it's a requirement.